.travis.yml

```diff
@@ -1,6 +1,10 @@
 sudo: false
 language: php
 
+env:
+  global:
+    - secure: "foo"
```

Comment:
@bighappyface 

You actually need to create GitHub token and ideally encrypt it, then replace the result here:

```
$ travis encrypt GITHUB_TOKEN=foo

Please add the following to your .travis.yml file:

  secure: "..."

Pro Tip: You can add it automatically by running with --add.
```

:information_desk_person: Do not use `--add`, it just messes up `.travis.yml` by rewriting it.